data/GA/incentives.json

```diff
@@ -131,14 +131,14 @@
     "amount": {
       "type": "percent",
       "number": 0.5,
-      "maximum": 500
+      "maximum": 1000
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
-      "en": "Rebate for 50% of cost up to $500 for ENERGY STAR® certified heat pump water heaters.",
-      "es": "Reembolso del 50% de los costos  para calentadores de agua con bomba de calor certificados ENERGY STAR, hasta $500 dólares."
+      "en": "Rebate for 50% of cost up to $100 for ENERGY STAR® certified heat pump water heaters.",
```

Comment:
$1000

Reply:
Changed back to $500 to match the official language (i.e. they're willing to subsidize the cost of a heat pump water heater by up to $500 assuming the homeowner spent at least $1000 overall)